docs/setup-cluster/deploy-cluster/slurm/slurm-requirements.rst

```diff
@@ -275,13 +275,13 @@ interacts with PBS, we recommend the following steps:
 .. _singularity-config-requirements:
 
 ************************************
- Singularity/Apptainer Requirements
+ Apptainer/Singularity Requirements
 ************************************
 
-Singularity/Apptainer is the recommended container runtime for Determined on HPC clusters. Apptainer
+Apptainer/Singularity is the recommended container runtime for Determined on HPC clusters. Apptainer
 is a fork of Singularity 3.8 and provides both the ``apptainer`` and ``singularity`` commands. For
 purposes of this documentation, you can consider all references to Singularity to also apply to
```

Comment:
> purposes of this documentation, you can consider all references to Singularity to also apply to Apptainer

I would have thought we would have reversed this statement.  

Previously we always just said "Singularity" and it was intended to mean both.   Did we change all instance of "Singularity" to "Apptainer" such that we should reverse the sentence?

Reply:
But not all Apptainer commands apply to Singularity.